releasenotes/notes/fix-mapper-bug-8eba9d219595b8e1.yaml

```
@@ -0,0 +1,7 @@
+---
+features:
```

```suggestion
fixes:
```
I think this should be 'fixes' right as it address a bug and is not a new feature

---

Thanks. My bad.